docs/guides/built-ins/actions/assignCodeAuthorReviewer/example.yml

```
@@ -0,0 +1,8 @@
+api-version: reviewpad.com/v3.x
+
+workflows:
+  - name: assign-reviewer
+    if:
+      - '!$isWaitingForReview() && !isDraft()'
+    then:
+      - $assignCodeAuthorReviewer()
```

Maybe we could present some arguments in this example since we don't in https://github.com/reviewpad/docs/pull/194/files#diff-119144fa4cd4ae9c48e314d8aa4ce22f664694e46a66dc43500e30de11dd4587R29
```suggestion
      - $assignCodeAuthorReviewer(2, ["john", "marie"])
```